databricks_cli/unity_catalog/ext_loc_cli.py

```diff
@@ -39,6 +39,10 @@
               help='Path URL for the new external location')
 @click.option('--storage-credential-name', default=None,
               help='Name of storage credential to use with new external location')
+@click.option('--read-only/--no-read-only', is_flag=True, default=None,
```

Comment:
Wow!  I'm thrilled that when the `data` dict has fields like `'read_only': None`, the JSON sent has `"read_only": null` and then the server sees that field omitted.  Party on, Garth....

Reply:
Null magic!